### PR TITLE
Add requirements around certificate validity in the absence of Authen…

### PR DESCRIPTION
**Pull request by rgnote**
Clarify requirements around certificate validity if AuthenticTimestamp is not available.

@gokarnm @SteveLasker @roywill

Signed-off-by: rgnote <5878554+rgnote@users.noreply.github.com>

### CODE REVIEW

**Comment by SteveLasker** [comment]
adding @shizhMSFT

File: trust-store-trust-policy-specification.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ The following table shows the resultant validation action, either *enforced* (ve
 
 **Authenticity** : Guarantees that the artifact was signed by an identity trusted by the verifier. Its definition does not include revocation, which is when a trusted identity is subsequently untrusted because of a compromise.
 
-**Authentic timestamp** : Guarantees that the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature must be treated as valid when a certificate is revoked, if the certificate was revoked after the signature was generated. In the absence of a authentic timestamp, signatures are considered invalid after certificate expires, and all signatures are considered revoked when a certificate is revoked.
+**Authentic timestamp** : Guarantees that the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature must be treated as valid or not based on whether the signature was generated before or after the certificate revocation. In the absence of an Authentic Timestamp, a signature is considered invalid if the signing certificate or chain is either expired or revoked.
```
Comment (shizhMSFT) [inline trust-store-trust-policy-specification.md]: To reduce ambiguity in English.
```suggestion
**Authentic timestamp** : Guarantees that the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature must be treated as valid or invalid based on whether the signature was generated before or after the certificate revocation. In the absence of an Authentic Timestamp, a signature is considered invalid if the signing certificate or chain is either expired or revoked.
```

File: signature-specification.md [hunk 2]
```diff
@@ -229,6 +229,7 @@ The `keyUsage` extension MUST be present and MUST be marked critical. Bit positi
 1. Any certificate in the certificate chain MUST NOT use SHA1WithRSA and ECDSAWithSHA1 signatures.
 1. Only Basic Constraints, Key Usage, and Extended Key Usage extensions of X.509 certificates are honored. For rest of the extensions, Notary MUST fail open i.e. they MUST NOT be evaluated or honored.
 1. The certificates in the certificate chain MUST be valid at signing time. Notary MUST NOT enforce validity period nesting, i.e the validity period for a given certificate may not fall entirely within the validity period of that certificate's issuer certificate.
+1. In the absense of an Authentic Timestamp, each and every certificate in the signature i.e. signing certificate, intermediate certificates, and the root certificate must be valid i.e. not expired at the time of signature verification.
```
Comment (gokarnm) [inline signature-specification.md]:
```suggestion
1. In the absence of an Authentic Timestamp, each and every certificate in the certificate chain i.e. signing certificate, intermediate certificates, and the root certificate must be valid i.e. not expired at the time of signature verification.
```

**Review by gokarnm — CHANGES_REQUESTED** [review]

**Review by gokarnm — APPROVED** [review]
LGTM!

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Comment by gokarnm** [comment]
@roywill would appreciate your review for this PR.

**Review by roywill — APPROVED** [review]

**Pull request merged**